web/samples/helmet.html

```diff
@@ -95,13 +95,17 @@
 
         const cameraEntity = Filament.EntityManager.get().create();
         this.camera = engine.createCamera(cameraEntity);
-        this.camera.setExposure(32, 1/125.0, 100);
+
+        const colorGrading = Filament.ColorGrading.Builder()
+            .toneMapping(Filament.ColorGrading$ToneMapping.ACES)
```

Comment:
You could add a vignette if you want to make it obvious

Reply:
Oh, actually I need to add bindings for that, it's a View API not a ColorGrading API.  Will do so next.